src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The excavator stopped working");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative and indicate that it is an `Excavator` that stopped working, for example: 'Excavator stopped its work.'